src/concourse/pipelines/infrastructure/ovs/docker_packer_pulumi_pipeline.py

```diff
@@ -71,47 +60,23 @@ def build_ovs_pipeline() -> Pipeline:
         ],
     )
 
-    dcind_resource = AnonymousResource(
-        type=REGISTRY_IMAGE,
-        source=RegistryImage(repository="mitodl/concourse-dcind", tag="latest"),
-    )
-
-    # TODO MD 20230110
-    # May be able to convert this to something that uses 'concourse.lib.tasks.container_build_task'
     docker_build_job = Job(
         name="build-ovs-image",
         build_log_retention={"builds": 10},
         plan=[
             GetStep(get=ovs_rc_repo.name, trigger=True),
-            TaskStep(
-                task=Identifier("build-ovs-container"),
-                privileged=True,
-                config=TaskConfig(
-                    platform=Platform.linux,
-                    image_resource=dcind_resource,
-                    inputs=[Input(name=ovs_rc_repo.name)],
-                    outputs=[Output(name=ovs_rc_repo.name)],
-                    run=Command(
-                        path="/bin/entrypoint.sh",
-                        args=[
-                            "bash",
-                            "-ceux",
-                            textwrap.dedent(
-                                f"""mount -t cgroup -o none,name=systemd cgroup /sys/fs/cgroup/systemd
-                                cd {ovs_rc_repo.name}
-                                chmod 777 .
-                                docker build . -f Dockerfile --target=production -t ovs-app:latest
-                                FINAL_IMG_ID="$(docker images | grep "ovs-app" |  tr -s ' ' | cut -d ' ' -f 3)"
-                                docker save "${{FINAL_IMG_ID}}" -o ovs-app.tar"""
-                            ),  # noqa: WPS355
-                        ],
-                    ),
-                ),
+            container_build_task(
+                inputs=[Input(name=ovs_rc_repo.name)],
+                build_parameters={
+                    "CONTEXT": ovs_rc_repo.name,
+                    "TARGET": "production",
```

Comment:
Does this need to be hardcoded?

Reply:
Yeah, refers to the target in the OVS dockerfile: 

https://github.com/mitodl/odl-video-service/blob/master/Dockerfile#L47-L52

If you don't specify it, you end up with the :development version" image, which we (devops) have no use for.